baselines/pfsp/pfsp_gpu_cuda.c

```diff
@@ -0,0 +1,570 @@
+/*
+  Single CUDA GPU B&B to solve Taillard instances of the PFSP in C.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+#include <limits.h>
+#include <getopt.h>
+#include <time.h>
+#include <math.h>
+//#include <cuda.h>
+#include <cuda_runtime.h>
+
+#include "lib/c_bound_simple.h"
+#include "lib/c_bound_johnson.h"
+#include "lib/c_taillard.h"
+#include "evaluate.h"
+
+
+/*******************************************************************************
+Implementation of PFSP Nodes.
+*******************************************************************************/
+
+// BLOCK_SIZE, MAX_JOBS and struct Node are defined in parameters.h
```

Comment:
C'est un peu étrange de séparer la fonction `initRoot` de `struct Node`, mais OK pour l'instant. On règlera ça plus tard.

Reply:
I added a comment TO DO in order to remind me to create the separate files for the nodes and the pools. As you did for Chapel.